src/js/server/models/Hacker.js

```diff
@@ -141,7 +145,15 @@ const Hacker = module.exports = db.define('hacker', {
   }
 });
 
+Hacker.TooYoungError = class TooYoungError extends Error { };
+
 Hacker.upsertAndFetchFromMlhUser = function (mlhUser) {
+  if (moment(mlhUser.date_of_birth).isAfter(under18Cutoff)) {
+    console.log('User is too young');
```

Comment:
If we want to leave this log statement in, do we also want to display the user's name / ID for a little more diagnostic information?